content/ru/service-mesh.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Сервис-меш
+status: Completed
+category: technology
+tags: ["networking", "", ""]
+---
+
+В [микросервисном](/ru/microservices-architecture) мире приложения разбиваются на множество небольших [сервисов](ru/service), которые взаимодействуют по сети. 
+Прямо как ваш домашний Wi-Fi, компьютерные сети по своей природе ненадёжные, небезопасные и нередко медленные. 
+Сервис-меши (service meshes) отвечают на эти вызовы, управляя трафиком (то есть взаимодействием) между сервисами 
+и добавляя фичи [надёжности](/ru/reliability), [наблюдаемости](/ru/observability) и безопасности единообразно для всех сервисов.
+
+## Какую проблему решает 
+
+После перехода на микросервисную архитектуру инженеры имеют дело с сотнями, возможно, даже с тысячами отдельных сервисов, 
+которым необходимо взаимодействовать между собой. 
+Это означает, что по сети туда и обратно проходит много трафика. 
+Кроме того, отдельные приложения могут шифровать трафик во исполнение нормативных требований, 
+экспортировать единые метрики для операционных команд или предоставлять детальную информацию о трафике для диагностики проблем. 
+Встраивание этих функций в каждое приложение осложнит взаимодействие инженерных команд и замедлит развёртывание новых фичей. 
```

Comment:
Well, is it legit to refer to the grammatical cases' table [here](https://ru.wiktionary.org/wiki/%D1%84%D0%B8%D1%87%D0%B0)? :-D
```suggestion
Встраивание этих функций в каждое приложение осложнит взаимодействие инженерных команд и замедлит развёртывание новых фич. 
```